x-pack/plugins/canvas/public/components/workpad_app/workpad_app.component.tsx

```diff
@@ -5,24 +5,31 @@
  * 2.0.
  */
 
-import React, { FC, MouseEventHandler, useRef, useCallback } from 'react';
+import React, { FC, MouseEventHandler, useRef, useCallback, useEffect } from 'react';
 import PropTypes from 'prop-types';
+import { CANVAS } from '../../../i18n';
 import { Sidebar } from '../sidebar';
 import { Toolbar } from '../toolbar';
 import { Workpad } from '../workpad';
 import { WorkpadHeader } from '../workpad_header';
 import { CANVAS_LAYOUT_STAGE_CONTENT_SELECTOR } from '../../../common/lib/constants';
-import { CommitFn } from '../../../types';
+import { CanvasWorkpad, CommitFn } from '../../../types';
 
 export const WORKPAD_CONTAINER_ID = 'canvasWorkpadContainer';
 
 export interface Props {
   deselectElement?: MouseEventHandler;
   isWriteable: boolean;
+  workpad: CanvasWorkpad;
 }
 
-export const WorkpadApp: FC<Props> = ({ deselectElement, isWriteable }) => {
+export const WorkpadApp: FC<Props> = ({ deselectElement, isWriteable, workpad }) => {
   const interactivePageLayout = useRef<CommitFn | null>(null); // future versions may enable editing on multiple pages => use array then
+  const workpadTitle = useRef<HTMLHeadingElement>(null); // future versions may enable editing on multiple pages => use array then
+
+  useEffect(() => {
+    workpadTitle.current?.focus();
+  }, [workpadTitle]);
```

Comment:
This works great, especially until https://github.com/elastic/kibana/issues/38980 gets resolved. Once the fix for that issue is in though, I imagine we would want to get rid of this - perhaps we should make a comment calling that out?  🤷 

@1Copenut - figured I should double check with you that pulling the focus here is the right move. Basically, it helps pull the focus to **inside of Canvas** so all the items in the top Kibana navigation bar get skipped.

Reply:
@Heenawter Great call, and thank you for the heads up!

I agree this is the right action to take for now, and if your team has a preferred way to leave a self-reminder, let's do that too.

I'm meeting up with @smith next week to discuss a longer-term approach for (hopefully) a universal routing strategy. Basically it involves re-using the `EuiScreenReaderLive` component with an update I made in v60.1.0. The plan will be to use that component right at the top of the source order, set focus on it on route change, and announce the page title in a live region.

That is a long-tail item, so setting the focus on the H1 is a good mid-term strategy, and one that the Discover team has implemented in #134942.

Reply:
Amazing, thank you for the thorough response! 💪